src/worker.spec.ts

```diff
@@ -0,0 +1,15 @@
+import * as worker from "./worker";
+
+describe("worker", (): void => {
```

Comment:
TS assumes `void` if you don't return anything so you can drop the explicit `void` return type from these funcs if you like

Reply:
I dug into this, even though our `tsconfig.json` doesn't set it, I am getting warnings from the ESLint TypeScript plugins for `"explicit-function-return-type": "warn"`. Hence why I've been declaring the return type. https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/src/configs/recommended.json#L10